core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -81,6 +81,7 @@
     public static final String PREFERRED_WRITE_PARTITIONING_MIN_NUMBER_OF_PARTITIONS = "preferred_write_partitioning_min_number_of_partitions";
     public static final String SCALE_WRITERS = "scale_writers";
     public static final String TASK_SCALE_WRITERS_ENABLED = "task_scale_writers_enabled";
+    public static final String MAX_WRITER_NODES_COUNT = "max_writer_nodes_count";
```

Comment:
I think this commit doesn't make sense w/o the last commit. So, maybe its better to merge them together?

Reply:
In the past there was exactly same situation and I was asked by maintainer to create a separate commit that was introducing a parameter to the session. This is why I put it in another commit.